share/lib/python/neuron/rxd/region.py

```diff
@@ -149,9 +149,7 @@ def _ecs_initalize(self):
             for sec in self._overlap:
                 for seg in sec:
                     (x,y,z) = species._xyz(seg)
-                    #TODO: Returns none, causing an error.
-                    node_idx = s().index_from_xyz(x,y,z)
-                    self.ecs_location_index[sid][seg_idx] = node_idx if node_idx else -1
+                    self.ecs_location_index[sid][seg_idx] = s().index_from_xyz(x,y,z)
```

Comment:
should it just be if node_idx is not None?

My concern is: what happens if the extracellular space isn't begin enough for the geometry and there is no mapping?

Reply:
I think we moved the check here; https://github.com/neuronsimulator/nrn/blob/5aac04b987ef49131f6fb7a84b02b67178934627/share/lib/python/neuron/rxd/species.py#L304
So we either get a non-negative number or an Exception.

Reply:
True.